modules/rosa-sdpolicy-platform.adoc

```diff
@@ -19,34 +19,36 @@ It is critical that customers have a backup plan for their applications and appl
 ====
 
 Application and application data backups are not a part of the {product-title} service.
-All Kubernetes objects and persistent volumes (PVs) in each {product-title} cluster are backed up to facilitate a prompt recovery in the unlikely event that a cluster becomes irreparably inoperable.
+The following table outlines the cluster backup policy.
 
-The backups are stored in a secure object storage, or multiple availability zone, bucket in the same account as the cluster.
-Node root volumes are not backed up, as Red Hat CoreOS is fully managed by the {product-title} cluster and no stateful data should be stored on a node's root volume.
+//Verify if the corresponding tables in policy-incident.adoc and rosa-policy-incident.adoc also need to be updated.
```

Comment:
The tables should be identical. Is it possible to break this table out into a separate module for reuse, and to prevent modifying multiple tables?

Reply:
Thanks for the suggestion, Will! Because of the way the current modules are structured, pulling the table into a separate module would change the flow in each of these pages. 

In rosa-policy-incident.adoc, _Backup and recovery_ would move below _Cluster capacity_. 

In policy-incident.adoc (OSD), _Backup and recovery_ would move below _Cluster capacity_. 

And in rosa-sdpolicy-platform.adoc, the title would change from _Cluster backup policy_ to _Backup and recovery_, and this one would be bumped below quite a few sub-sections. This one also includes a note, so it would be a bit more challenging to bring this one in the same module as the other two.

I'm open to tinkering with it, but wanted to see if it makes sense with the other considerations.

Reply:
Yeah, no worries. Just trying to make things easier...lol. Tables LGTM